cdap-ui/app/cdap/components/DataPrep/Directives/MapToTarget/index.js

```diff
@@ -14,36 +14,365 @@
  * the License.
  */
 
-import React, { Component } from 'react';
+import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
 import T from 'i18n-react';
+import { makeStyles } from '@material-ui/core';
+import Input from '@material-ui/core/Input';
+import LinearProgress from '@material-ui/core/LinearProgress';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import { UncontrolledTooltip } from 'reactstrap';
 import classnames from 'classnames';
+import { preventPropagation, connectWithStore } from 'services/helpers';
+import { setPopoverOffset } from 'components/DataPrep/helper';
+import DataPrepStore from 'components/DataPrep/store';
+import {
+  execute,
+  setError,
+  loadTargetDataModelStates,
+  saveTargetDataModelStates,
+  setTargetDataModel,
+  setTargetModel
+} from 'components/DataPrep/store/DataPrepActionCreator';
+
+const useStyles = makeStyles(theme => ({
+  secondLevelPopover: {
+    width: '300px !important',
```

Comment:
You can probably use id to override the rule. ie
root: {
  '& #map-to-target-second-level-popover': {
    width: '300px',
  }
}

Reply:
It does not work.

Reply:
Some data model and model names are too long so they do not fit into the width of `200px` that is default width for second level popover. Anyway, removed it.